PokeAlarm/Load.py

```diff
@@ -113,10 +113,10 @@ def parse_filter_section(section):
     defaults = section.pop('defaults', {})
     default_dts = defaults.pop('custom_dts', {})
     filter_set = OrderedDict()
-    for name, settings in section.pop('filters', {}).iteritems():
-        settings = dict(defaults.items() + settings.items())
-        local_dts = dict(default_dts.items()
-                         + settings.pop('custom_dts', {}).items())
+    for name, settings in section.pop('filters', {}).items():
+        settings = dict(list(defaults.items()) + list(settings.items()))
```

Comment:
Have you verified this works right? I'm not sure of the behavior of generator -> `List<Tuple>` -> dict

Reply:
Looks like it might be okay - dict seems to be able to take a list of tuples ok

Might be more efficient/cleaner to use [`itertools.chain`](https://docs.python.org/3/library/itertools.html#itertools.chain) instead of converting though